CreeDictionary/API/search.py

```diff
@@ -14,25 +14,28 @@
     Optional,
     Set,
     Tuple,
+    TypeVar,
     Union,
     cast,
 )
 
 import attr
 from attr import attrs
 from cree_sro_syllabics import syllabics2sro
-from CreeDictionary import hfstol as temp_hfstol
 from django.conf import settings
 from django.db.models import Q
 from sortedcontainers import SortedSet
+from typing_extensions import Protocol
+
+from CreeDictionary import hfstol as temp_hfstol
```

Comment:
I presume we'd want to generalize the reference to the dictionary, to `MorphoDict` rather than `CreeDictionary`.